examples/colourpicker-histogram.py

```diff
@@ -0,0 +1,48 @@
+from dash import Dash, dcc, html, Input, Output
+import plotly.graph_objects as go
+import plotly.express as px
+import pandas as pd
+import dash_daq as daq
+import plotly.io as pio
+
+df = px.data.gapminder()
+df = df.loc[df['year']==1952]
+
+pio.templates.default = "ggplot2"
+
+app = Dash(__name__)
+
+picker_style = {"display": "inline-block", "margin": 10}
+
+app.layout = html.Div(
+    [
+        html.H2("Interactive Bar Colour Control"),
+        dcc.Loading(dcc.Graph(id="colourpicker-histogram-x-graph"), type="circle"),
+        daq.ColorPicker(
+            id="colourpicker-histogram-x-color",
+            label="Color of Bars",
+            size=164,
+            value=dict(hex="#564AE6")
+        ),
+    ]
+)
+
+
+@app.callback(
+    Output("colourpicker-histogram-x-graph", "figure"), Input("colourpicker-histogram-x-color", "value"),
+)
+
+
+def update_points_color(color):
```

Comment:
Ann's suggestion putting the app to sleep for 2 seconds will make the loading component (spinner) visible. You can add this line of code right after defining the callback and before you start building the figure:
`time.sleep(2)`

Done forget to import time at the very top of the fily